e2e/test/iothub/command/CommandE2ETests.cs

```diff
@@ -120,44 +121,45 @@ public static async Task DigitalTwinsSendCommandAndVerifyResponseAsync(string de
             digitalTwinClient.Dispose();
 #endif
             logger.Trace($"{nameof(DigitalTwinsSendCommandAndVerifyResponseAsync)}: Command status: {statusCode}.");
-            Assert.AreEqual(200, statusCode, $"The expected response status should be 200 but was {statusCode}");
-            Assert.AreEqual(responseExpected, payloadReceived, $"The expected response payload should be {responseExpected} but was {payloadReceived}");
+            statusCode.Should().Be(200, "The expected response status should be 200.");
+            payloadReceived.Should().Be(responseExpected);
         }
 
         public static async Task<Task> SetDeviceReceiveCommandAsync(DeviceClient deviceClient, string componentName, string commandName, MsTestLogger logger)
         {
             var commandCallReceived = new TaskCompletionSource<bool>();
 
-            await deviceClient.SubscribeToCommandsAsync(
-                (request, context) =>
-                {
-                    logger.Trace($"{nameof(SetDeviceReceiveCommandAsync)}: DeviceClient command: {request.CommandName}.");
-
-                    try
+            await deviceClient
+                .SubscribeToCommandsAsync(
+                    request =>
                     {
-                        var valueToTest = request.GetData<ServiceCommandRequestObject>();
-                        if (string.IsNullOrEmpty(componentName))
+                        logger.Trace($"{nameof(SetDeviceReceiveCommandAsync)}: DeviceClient command: {request.CommandName}.");
+
+                        try
                         {
-                            Assert.AreEqual(null, request.ComponentName, $"The expected component name should be null but was {request.ComponentName}");
+                            var valueToTest = request.GetPayload<ServiceCommandRequestObject>();
+                            if (string.IsNullOrEmpty(componentName))
+                            {
+                                request.ComponentName.Should().BeNull();
```

Comment:
So `componentName` could be whitespace (legally) but we'd still expect `request.ComponentName` to be null?

Reply:
This can be simplified, for sure.